src/configurations/destinations/postgres/ui-config.json

```diff
@@ -59,6 +59,69 @@
           "footerNote": "Default will be the source name",
           "immutable": true
         },
+        {
+          "type": "checkbox",
+          "label": "SSH Connection",
+          "value": "useSSH",
+          "default": false,
+          "footerNote": "Establish ssh tunnel connection",
+          "featureFlag": "WAREHOUSE_SSH_TUNNELLING"
```

Comment:
Since the following "block" of fields have `useSSH` as a preRequisite we only need to control this `Switch` using a featureFlag.

Reply:
Rename `featureFlag` to `preRequisiteFeatureFlag`